iotedgedev/envvars.py

```diff
@@ -209,6 +209,26 @@ def save_envvar(self, key, value):
             self.output.error(f("Could not update the environment variable {key} in file {dotenv_path}"))
             sys.exit(-1)
 
+    def get_registries(self):
+        registries = {}
+        self.CONTAINER_REGISTRY = {}
```

Comment:
Extremely minor, consider renaming self.CONTAINER_REGISTRY to something like CONTAINER_REGISTRY_MAP or CONTAINER_REGISTRIES_BY_SERVER, etc. Current name could be interpreted as an object describing a single registry.